### PR TITLE
VR > Month View > Dates should link to day view

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/133035

🎥 https://www.loom.com/share/1cbf2df2fb2541e6bd7291f95dc38a4c

- Add past day classes
- Add links to day date number when there are events.
- Add data attributes.
- Add a11y data.
- Add template var for day link.
- Fix `texdomain` for some templates.

### CODE REVIEW

**Comment by paulmskim** [comment]
@juanfra what do past featured events look like? How about multiday events?

File: src/views/v2/month/calendar-body/day.php [hunk 1]
```diff
@@ -63,8 +68,17 @@ class="tribe-events-calendar-month__day-cell tribe-events-calendar-month__day-ce
 		tabindex="-1"
 	>
 		<h3 class="tribe-events-calendar-month__day-date tribe-common-h6 tribe-common-h--alt">
-			<span class="tribe-common-a11y-visual-hide">X events, </span>
-			<time datetime="YYYY-MM-DD">
+			<span class="tribe-common-a11y-visual-hide">
+				<?php echo esc_html(
+								sprintf(
```
Comment (bordoni) [inline src/views/v2/month/calendar-body/day.php]: Indentation is odd here.

File: src/views/v2/month/calendar-body/day.php [hunk 2]
```diff
@@ -90,15 +104,27 @@ class="tribe-events-calendar-month__mobile-events-icon tribe-events-calendar-mon
...
-			<span class="tribe-common-a11y-visual-hide">X events, </span>
-			<time datetime="YYYY-MM-DD">
-				<a
-					<?php /* @todo @fe connect this to the day view link if the day has events */ ?>
-					href="#link-to-day-view-if-it-has-events"
-					class="tribe-events-calendar-month__day-date-link"
-				>
+			<span class="tribe-common-a11y-visual-hide">
+				<?php echo esc_html(
+								sprintf(
```
Comment (bordoni) [inline src/views/v2/month/calendar-body/day.php]: Odd indentation.

File: src/views/v2/month/calendar-body/day.php [hunk 3]
```diff
@@ -90,15 +104,27 @@ class="tribe-events-calendar-month__mobile-events-icon tribe-events-calendar-mon
...
+									)
+								)
+							);
+				?>,
+			</span>
+			<time datetime="<?php echo esc_attr( $day['date'] ); ?>">
+				<?php if ( ! empty( $day['events'] ) ) : ?>
+					<a
+						href="<?php echo esc_url( $day['day_url'] ); ?>"
+						class="tribe-events-calendar-month__day-date-link"
```
Comment (bordoni) [inline src/views/v2/month/calendar-body/day.php]: This needs to be a `data-js="tribe-events-view-link"` link

File: src/Tribe/Views/V2/Views/Month_View.php [hunk 4]
```diff
@@ -272,6 +273,8 @@ static function ( $event ) use ( $date_object ) {
 
 			$start_of_week = get_option( 'start_of_week', 0 );
 
+			$day_url = Plugin::instance()->getLink( 'day', $day_date );
```
Comment (bordoni) [inline src/Tribe/Views/V2/Views/Month_View.php]: This is the old way of setting up the URLs, we want to use the method:
https://github.com/moderntribe/the-events-calendar/pull/2664/files#diff-d0c94939e63f4b6608c4f83c2e99ec9cR28

File: src/Tribe/Views/V2/Views/Month_View.php [hunk 5]
```diff
@@ -283,6 +286,7 @@ static function ( $event ) use ( $date_object ) {
 				'multiday_events'  => $day_stack,
 				'found_events'     => $day_found_events,
 				'more_events'      => $more_events,
+				'day_url'          => $day_url,
```
Comment (bordoni) [inline src/Tribe/Views/V2/Views/Month_View.php]: I do like that approach. Wiill merge into mine

Comment (juanfra) [reply src/Tribe/Views/V2/Views/Month_View.php]: Awesome! yeah it's what Luc4 suggested we want to start doing.

**Pull request merged**